src/ansible_navigator/runner/ansible_config.py

```diff
@@ -5,7 +5,7 @@
 from typing import Optional
 from typing import Tuple
 
-from ansible_runner import get_ansible_config
+from ansible_runner import get_ansible_config  # type: ignore
```

Comment:
Instead of adding a catch-all ignore, only ignore specific error codes. It'd look like `# type: ignore[mistake-id]`. This way you wouldn't accidentally ignore extra problems that may arise in the future.

Reply:
this is b/c type hints aren't available for ansible_runner

```
error: Skipping analyzing "ansible_runner": found module but no type hints or library stubs
```

From the docs: 

```
If you are unable to find any existing type hints nor have time to write your own, you can instead suppress the errors. All this will do is make mypy stop reporting an error on the line containing the import: the imported module will continue to be of type Any.

To suppress a single missing import error, add a # type: ignore at the end of the line containing the import.
```

Reply:
Unresolving. This is not what I am asking about. I'm asking to append a specific error code, just like you do with pylint. You don't ignore every linting violation type possible when you see a specific one, you disable just that one thing. Same with mypy.

Reply:
I hope this document will help you understand what I mean: 
https://mypy.readthedocs.io/en/stable/error_codes.html#error-codes

Reply:
This was ignored because it is a 3rd party library that does not have hints.

I think this documentation may be more relevant that the one you suggested: https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-type-hints-for-third-party-library

Reply:
Interesting, does it not have its own code? If not, at least add code comments explaining why this is necessary. In fact, code comments with reasoning are needed every time you ignore a linting problem. Otherwise, it's hard to figure it out in the future.

Reply:
@webknjaz If you are ware of a better way to handle this, please let me know.

Reply:
Looks like mypy is not configured to show error codes. Try adding `--show-error-codes` to see them.

Reply:
Looks like you need
```suggestion
from ansible_runner import get_ansible_config  # type: ignore[import]
```
per https://mypy.readthedocs.io/en/stable/error_code_list.html#check-that-import-target-can-be-found-import

Reply:
OOoooo..... if that works I'll change it! cool.

Reply:
I love this, other instances will get cleaned up with this: https://github.com/ansible/ansible-navigator/issues/638

thanks